qemu/tests/scsi_mq_check.py

```diff
@@ -0,0 +1,61 @@
+import logging
+import re
+
+from virttest import error_context
+from virttest import utils_test
+from virttest import utils_misc
+
+
+def get_mq_from_guest(session, params, cmd, pattern):
+    """
+    Get multi queue number from guest.
+
+    :param session: VM session
+    :param params: Dictionary with the test parameters
+    :param cmd: command to get queue info
+    :param pattern: Get the queue number from queue info
+    """
+    status, output = session.cmd_status_output(cmd)
+    if status != 0:
+        logging.error("cmd running failed with output %s" % output)
```

Comment:
I think if cmd is failed, set value for queues (e.g. 0) and then return it directly. No need to get queues in the next codes.

Reply:
Hi xuwei, I updated the patch, could you help to review it again?